.github/workflows/build.yaml

```
@@ -109,3 +109,14 @@ jobs:
       sha: ${{ inputs.sha }}
       date: ${{ inputs.date }}
       package-name: dask_cudf
+  pandas-tests:
+    # run the Pandas unit tests using main branch
```

This might address @ajschmidt8's request?
```suggestion
    # run the Pandas unit tests using main branch
    if: ${{ inputs.branch == 'nightly' }}
```

---

However, if we're only running this with nightlies, then we should move this whole thing to `test.yaml`. The entire `build.yaml` workflow has to complete before the `test.yaml` workflow begins, and we don't want to block normal cudf tests if this job fails or force them to wait if this takes a long time.

---

I agree with this. We should ideally move these tests to `test.yaml`